docs/edgeql/sets.rst

```diff
@@ -248,6 +248,75 @@ Use the :eql:op:`union` operator to merge two sets.
   db> select {1, 2} union {3.1, 4.4};
   {1.0, 2.0, 3.1, 4.4}
 
+Finding common members
+----------------------
+
+.. versionadded:: 3.0
+
+Use the :eql:op:`intersect` operator to find common members between two sets.
+
+.. code-block:: edgeql-repl
+
+    db> select {1, 2, 3, 4, 5} intersect {3, 4, 5, 6, 7};
+    {3, 5, 4}
+    db> select {'a', 'b', 'c', 'd', 'e'} intersect {'c', 'd', 'e', 'f', 'g'};
+    {'e', 'd', 'c'}
+
+If set members are repeated in both sets, they will be repeated in the set
+produced by :eql:op:`intersect` the same number of times they are repeated in
+both of the operand sets.
+
+.. code-block:: edgeql-repl
+
+    db> select {0, 1, 1, 1, 2, 3, 3} intersect {1, 3, 3, 3, 3, 3};
+    {1, 3, 3}
+
+In this example, ``1`` appears three times in the first set but only once in
+the second, so it appears only once in the result. ``3`` appears twice in the
+first set and five times in the second. Both ``3`` appearances in the first set
+are overlapped by ``3`` appearances in the second, so they both end up in the
+resulting set.
+
+
+.. note::
```

Comment:
My inclination is to drop these notes, since there's no real reason to think that ordering /would/ be preserved, and I worry that noting this might make people assume it is preserved in other cases we don't mention

Reply:
Makes sense. I removed those.